website/util/sanitize.py

```diff
@@ -74,6 +74,18 @@ def safe_unescape_html(s):
         '&lt;': '<',
         '&gt;': '>',
     }
-    for escape_sequence, character in safe_characters.items():
-        s = s.replace(escape_sequence, character)
+    if isinstance(s, dict):
+        return {
+            key: safe_unescape_html(value)
+            for (key, value) in s.iteritems()
+        }
+    if isinstance(s, list):
+        return [
+            safe_unescape_html(value)
+            for value in s
+        ]
+    if isinstance(s, basestring):
```

Comment:
Minor, but it might make sense to put this conditional before the others, since this is the common case. This improves readability and potentially saves unnecessary calls to `isinstance`